.github/workflows/platformDeploy.yml

```diff
@@ -346,10 +346,10 @@ jobs:
           aws s3 cp --acl public-read --content-type 'application/json' --metadata-directive REPLACE ${{ env.S3_URL }}/.well-known/apple-app-site-association ${{ env.S3_URL }}/.well-known/apple-app-site-association
           aws s3 cp --acl public-read --content-type 'application/json' --metadata-directive REPLACE ${{ env.S3_URL }}/.well-known/apple-app-site-association ${{env.S3_URL }}/apple-app-site-association
         env:
-          S3_URL: s3://${{ fromJSON(env.SHOULD_DEPLOY_PRODUCTION) && '' || 'staging-' }}expensify-cash
+          S3_URL: s3://${{ env.SHOULD_DEPLOY_PRODUCTION != 'true' && 'staging-' || '' }}expensify-cash
```

Comment:
The code was confusing to me before and is still confusing to me now. Is this code equivalent?
```
          S3_URL: s3://${{ env.SHOULD_DEPLOY_PRODUCTION != 'true' ? 'staging-' : '' }}expensify-cash
```

And if so, why not write that which IMO is a lot more clearer than the current code?

Reply:
GH actions syntax doesn't support that kind of ternary - https://docs.github.com/en/actions/learn-github-actions/expressions#example

Reply:
That's so bad.... seems people asked for it but they did not add it https://github.com/actions/runner/issues/409

Someone even pointed out the bug here https://github.com/actions/runner/issues/409#issuecomment-1013325196 

Fuck GH really

Reply:
😂